index.js

```diff
@@ -65,9 +77,9 @@ module.exports = (number, options) => {
 		return prefix + numberString + ' ' + UNITS[0];
 	}
 
-	const exponent = Math.min(Math.floor(Math.log10(number) / 3), UNITS.length - 1);
+	const exponent = Math.min(Math.floor(Math.log(number) / Math.log(options.binary ? 1024 : 1000)), UNITS.length - 1);
```

Comment:
I would prefer continuing to use `Math.log10` for the non-binary case.

Reply:
Done.